examples/clangd-contexts-ext/package.json

```
@@ -0,0 +1,73 @@
+{
+  "private": true,
+  "name": "theia-clangd-contexts-ext",
```

```suggestion
  "name": "cdtcloud-clangd-contexts-ext",
```

I think this would have ripple effects in other places, too (e.g. `npm` scripts).